src/errors/remote-quote-error.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const BaseError = require('extensible-error')
+
+const { codes } = require('../lib/ilp-errors')
+
+class RemoteQuoteError extends BaseError {
+  constructor (message) {
+    super(message)
+
+    // TODO: Is this the right error code? Maybe we should pass on the error we
+    //   we received where possible?
+    this.ilpErrorCode = codes.T00_INTERNAL_ERROR
```

Comment:
I'm not sure these errors will be temporary if other connectors just stop supporting ILQP

Reply:
Good point - what's the right error code here? Or should we add one?

(I ran into a few cases where I was like: might be good to add a code for this - or make an existing code less specific.)